### ISSUE TITLE
Chapter 8 page 283 Express Node 

### ISSUE THREAD

**Issue by mariusErpenbach** [PacktPublishing/Full-Stack-React-TypeScript-and-Node#24]
Working on the expressapp.mjs :
The CURL command for the terminal is not working anymore. 

we set up all the middleware -> then we set up a route object and set up the middleware routes -> then we want to test our POST method with curl, but its not working as intended.

This is the curl code we should use:
```
curl --header "Content-Type: application/json" --request POST
 --data `{"userid":"1","message":"hello"}` "http://localhost:8000/c"
```

this error message in the terminal output:
```
userid:1: command not found
curl: no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information
```
note: im running my express Node server on another terminal. The CURL worked for a previous section of the chapter where I also had to test a POST url request.

**Comment by jsoneaday** [comment]
Your curl command is broken. Your sample is actually written as two lines instead of one. It needs to be a single line.

**Comment by mariusErpenbach** [comment]
Excuse me for that, I broke the command in multiple lines as it was printed on page 269, but actually i am using it in one line, the previous example with node but without express was working with the curl command as intended, just for the express example its not working.

**Comment by mariusErpenbach** [comment]
yes, the route http://localhost:8000/a leads me to the route "/a" as intended, in the browser. My server is running on a second terminal, so im sure the server is running.

**Comment by mariusErpenbach** [comment]
I have also tried to change the package.json file to start the expressApp.mjs with npm start, so im 100% the server is running, as any change even restarts the server.
